src/manifolds/ProductManifold.jl

```diff
@@ -42,12 +42,22 @@ end
 
 const PRODUCT_BASIS_LIST_CACHED = [CachedBasis]
 
-"""
-    ProductMetric <: Metric
+@doc raw"""
+    ProductMetric{<:(Tuple{N,Metric} where N)} <: Metric
```

Comment:
The parens aren't necessary

```suggestion
    ProductMetric{<:Tuple{N,Metric} where N} <: Metric
```